prestodb/hive3.1-hive/Dockerfile

```diff
@@ -65,4 +65,6 @@ EXPOSE 9000 9870
 # HIVE Metastore port
 EXPOSE 9083 10000
 
+EXPOSE 1180
```

Comment:
This is an addition for debugging via SSH Is this safe? It looks like is is listening on all interfaces

Reply:
This is same as previous version image. SOCKS proxy is required to connect to services running on the container. All the hive integration tests configuration requires this to be setup. EXPOSE command here doesn't really have any effect though as it acts just as documentation. docker-compose.yml maps the actual ports required.

Reply:
SGTM